src/accessors/internal_accessor.ts

```diff
@@ -434,6 +437,32 @@ export class InternalAccessor implements IManagementApiAccessor {
     return this.flowNodeInstanceService.getFlowNodeInstancesForProcessInstance(identity, processInstanceId);
   }
 
+  // Tasks
+  public async getAllSuspendedTasks(identity: IIdentity): Promise<DataModels.Tasks.TaskList> {
+    return this.taskService.getAllSuspendedTasks(identity);
```

Comment:
Die Verweise hier kann man dann z.B. wie folgt ändern:

```suggestion
    return this.flowNodeInstanceService.getAllSuspendedTasks(identity);
```